sklearn/datasets/base.py

```diff
@@ -242,6 +242,108 @@ def load_files(container_path, description=None, categories=None,
                  DESCR=description)
 
 
+def load_wine(return_X_y=False):
+    """Load and return the wine dataset (classification).
+
+            .. versionadded:: 0.18
+
+    The wine dataset is a classic and very easy multi-class classification
+    dataset.
+
+    =================   ==============
+    Classes                          3
+    Samples per class               [59,71,48]
```

Comment:
I'm unsure of what you mean? Do you mean that the [59,71,48] needs to line up center wise with the 3? Please clarify